numba_pandas/tests/test_pandas.py

```diff
@@ -0,0 +1,24 @@
+import numba
+import pandas
+
+from packaging import version
+
+# numba <0.46 requires explicit import extension
```

Comment:
No need to check this. We will work with latest version. No legacy support is required.

Reply:
It is just for demonstrating how to test with previous versions of Numba <0.46.
I will remove because now porting to Numba 0.46 is merged to master.